lms/djangoapps/grades/tasks.py

```diff
@@ -138,7 +146,7 @@ def _has_database_updated_with_new_score(
         if api_score is None:
             # Same case as the initial 'if' above, for submissions-specific scores
             return score_deleted
-        reported_modified_time = api_score.created_at
+        reported_modified_time = api_score['created_at']
```

Comment:
It's used in django serializers, [as seen in the edx-submissions api we're calling into](https://github.com/edx/edx-submissions/blob/master/submissions/api.py#L577-L625). Effectively, it's just a dict, but I was accessing it's attributes with dot notation.